generators/heroku/templates/pom-profile.xml.ejs

```diff
@@ -1,3 +1,27 @@
+            <dependencyManagement>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.elasticsearch.client</groupId>
+                        <artifactId>elasticsearch-rest-client</artifactId>
+                        <version>7.10.2</version>
```

Comment:
version in the In the bom ?
Or in the profile's maven property section?

Reply:
Sounds good to have it in the profile.